src/webgl/primitives.js

```diff
@@ -1203,4 +1203,318 @@ p5.RendererGL.prototype.line = function() {
   return this;
 };
 
+p5.RendererGL.prototype.bezierVertex = function() {
+  if (this.immediateMode._bezierVertex.length === 0) {
+    throw 'vertex() must be used once before calling bezierVertex()';
+  } else {
+    var w_x = [];
+    var w_y = [];
+    var w_z = [];
+    var t, _x, _y, _z, i;
+
+    t = parseFloat(0.0);
+
+    if (
+      this._lookUpTableBezier.length === 0 ||
+      this._lookUpTableBezier[0] !== this._pInst._curveDetail
+    ) {
+      this._lookUpTableBezier = [];
+      this._lookUpTableBezier.push(this._pInst._curveDetail);
+      var step = 1 / this._pInst._curveDetail;
```

Comment:
What happens to this while loop if the user sets [_curveDetail](https://github.com/processing/p5.js/blob/master/src/core/shape/curves.js#L400) to 0?

Reply:
@mlarghydracept should there be a minimum curveDetail (3 perhaps) or it should render nothing when curveDetail is set to 0?

Reply:
That seems like a good solution to me.